### DISCUSSION TITLE
192kHz

### DISCUSSION THREAD

**Discussion by ajb1970** [rern/rAudio#1289] [General]
I am running your excellent software on a RPi 3B with a DigiAMP+. The app reports that music is played at 44kHz, is it possible to up the quality to 192kHz which the amp is capable of?

Thanks

**Comment by NexusPenguin** [comment]
Hi,

The player is of course capable to output much more than 44.1Khz. 

It should do so out of the box...

At least it does here. 

Maybe you can tell us a little more about your settings ? 

Did you enable camilla DSP ? 

If I enable Camilla DSP I am stuck at 44.1Khz too. 

I am not if I do not enable it.

**Comment by ajb1970** [comment]
Thanks. 
I have not enabled any of the feature settings such as Camilla DSP.
The volume is controlled by the mixer device, FFMPEG is enabled, along with a large buffer, everything else is off.
The device is connected over the LAN to my NAS.
rAudio is the 20230310 build.

I think that's it.

**Comment by NexusPenguin** [comment]
Hi,

This is weird. 

I have applied the same settings as you did. 

My raudio is running on a Pi4 Rev. B with 4GB RAM. 

I use the USB output of the Pi4.

And using that one I have no problem outputting 96Khz...

**Comment by ajb1970** [comment]
Perhaps that's it. My RPi3B+ only has 1Gb. I'll try it with my RPi4.

Thanks again

**Reply by NexusPenguin** [reply]
Hi, 

Maybe even though I highly doubt it...

But still worth a try :-)

Please let us know...
